### PR TITLE
Advanced metrics - add some details about store size metrics

### PR DESCRIPTION
**Pull request by eve-bright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-358.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-358.surge.sh/aura/auradb/managing-databases/advanced-metrics)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-358.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-358.surge.sh/aura/auradb/managing-databases/advanced-metrics)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-358.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-358.surge.sh/aura/auradb/managing-databases/advanced-metrics)

File: modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc [hunk 1]
```diff
@@ -81,3 +81,19 @@ The possible aggregations are:
 * *Max* - The maximum value of the metric across all cluster members.
 * *Average* - The average value of the metric across all cluster members.
 * *Sum* - The sum of the metric across all cluster members.
+
+== Store size metrics 
+
+=== Resources tab
+The chart on the resources tab shows the allocated store size metric for the selected database either as a percentage of the available storage assigned for the database or as absolute values.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc:88]:
```suggestion

The chart on the _Resources_ tab shows the allocated store size metric for the selected database either as a percentage of the available storage assigned for the database or as absolute values. 
```

File: modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc [hunk 2]
```diff
@@ -81,3 +81,19 @@ The possible aggregations are:
...
 * *Sum* - The sum of the metric across all cluster members.
+
+== Store size metrics 
+
+=== Resources tab
+The chart on the resources tab shows the allocated store size metric for the selected database either as a percentage of the available storage assigned for the database or as absolute values. 
+
+=== Database tab
+
+The chart on the Database tab provides more detailed information with a chart that shows the store size and the portion of the allocated space that the database is actively utilizing. Both metrics are represented as percentages of the available storage assigned to the database.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc:92]:
```suggestion
The chart on the _Database_ tab provides more detailed information with a chart that shows the store size and the portion of the allocated space that the database is actively utilizing. 
Both metrics are represented as percentages of the available storage assigned to the database.
```

**Review by AlexicaWright — COMMENTED** [review]
Just some style suggestions.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-358.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-358.surge.sh/aura/auradb/managing-databases/advanced-metrics)

File: modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc [hunk 3]
```diff
@@ -81,3 +81,21 @@ The possible aggregations are:
...
+
+== Store size metrics 
+
+=== Resources tab
+
+The chart on the _Resources_ tab shows the allocated store size metric for the selected database either as a percentage of the available storage assigned for the database or as absolute values. 
+
+=== Database tab
+
+The chart on the _Database_ tab provides more detailed information with a chart that shows the store size and the portion of the allocated space that the database is actively utilizing.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/managing-databases/advanced-metrics.adoc:93]:
```suggestion
The _Database_ tab provides a chart that shows the store size and the portion of the allocated space that the database is actively utilizing. 
```

**Review by AlexicaWright — COMMENTED** [review]
Just a small suggestion for readability.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-358.surge.sh

Updated pages:
- [Advanced metrics](https://neo4j-docs-aura-358.surge.sh/aura/auradb/managing-databases/advanced-metrics)

**Review by AlexicaWright — APPROVED** [review]
LGTM 👍

**Pull request merged**